docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -277,6 +277,20 @@ To learn more, refer to [Datadog data source settings](/docs/plugins/grafana-dat
 
 {{< video-embed src="/media/docs/datadog/datadog-rate-limit.mp4" >}}
 
+### Microsoft SQL Server: Support for Azure Authentication (Service principal/MSI)
+
+<!-- Jev Forsberg, Oscar Kilhed -->
+
+_Generally available in all editions of Grafana_
+
+We've added support for Azure Authentication (Service principal/MSI) on our MS SQL plugin to authenticate and allow querying of content stored in SQL Managed Instance databases.
+
+Enable this feature by setting the `managed_identity_enabled` property to `true` under the `Azure` heading in your configuration file (/conf/<your_config_file>.ini). Next, in your Microsoft SQL Server data source configuration UI, under **Authentication**, select **Azure AD Authentication** in the drop-down. And finally, under **Azure Authentication Settings**, select either **Managed Identity** or **App Registration**, and enter the credentials accordingly.
+
+{{< figure src="media/docs/grafana/data-sources/screenshot-managed-identity-mssql.png" caption="Azure MSI Authentication" >}}
```

Comment:
In my local, I can't get the image to build, but I'm having trouble with my local builds right now. As long as it's building for you, it's probably fine. However when I looked at the image in our media assets, it seems really big. Maybe add a width constraint on this (say 750px). And having the required fields not filled in is distracting. Can you put dummy content in those fields to avoid having those errors showing in the image?

Reply:
You got it!